python/pyspark/ml/regression.py

```diff
@@ -743,6 +743,18 @@ def treeWeights(self):
         """Return the weights for each tree"""
         return list(self._call_java("javaTreeWeights"))
 
```

Comment:
+1

On Fri, 6 May 2016 at 09:34, Yanbo Liang <notifications@github.com> wrote:

> In python/pyspark/ml/regression.py
> <https://github.com/apache/spark/pull/12919#discussion_r62296980>:
>
> > @@ -743,6 +743,18 @@ def treeWeights(self):
> >          """Return the weights for each tree"""
> >          return list(self._call_java("javaTreeWeights"))
> >
>
> I vote for base TreeEnsembleModels.trees return a list of
> DecisionTreeRegressionModel and override this method for
> RandomForestClassificationModel. We should add docs to clarify it to
> developers.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/12919/files/6400e9e3d7e096b05475cca548e31aabca8cbaff#r62296980>
>